docs/n00b-overview.md

```diff
@@ -11,17 +11,19 @@ B --> C[Server01]
 B --> D[Server02]
 ```
 
-And end up with this in the documentation:
+And they are rendered into this and made part of the documentation:
 
 ![Flowchart](./img/n00b-firstFlow.png)
 
-Most of the stuff I need to visualize can be scripted in a similar way, with a varitety of different symbols and chart types available. Since the diagram source is text based, it can be part of production scripts (and other pieces of code). So less time needs be spent on documenting as a separate task.
+Most of the similar visuals that you might need to create can be scripted in a similar way, with a varitety of different symbols and chart types available. 
+Since the diagram source is text based, it can be part of production scripts (and other pieces of code). So less time needs be spent on documenting as a separate task.
 
-Comparing with Visio and similar applications, mermaid is a really fast way to create good visualizations. This is especially apparent when editing a complex visualisation, this could take me hours in a desktop application but takes minutes (or even less if generation has been scripted) with mermaid.
-
-With mermaid I can spend a fraction of the time I normally would spend, and instead automate the diagram generation and end up saving even more time. I love it!
+Comparing with Visio and similar applications, mermaid is a really fast way to create good visualizations. This is especially apparent when editing a complex visualisations, a process that usually takes hours in a desktop application, but only takes minutes (or even less if generation has been scripted) with mermaid.
 
+mermaid can potentially cut down the amount of time and effort spent on the process of creating diagrams, to a fraction of what you sually put in.  
 
 However, a lot of the mermaid documentation is geared to professional frontend developers, presuming a skill set which I simply do not have.
 
-I needed a really basic instruction. And here it is.
+If you need some basic instructions and introductions. And here are a few good places to start:
```

Comment:
I would personally remove . And and place a comma.
If you need some basic instructions and introductions, here are a few good places to start: